.github/workflows/pythonpackage.yml

```diff
@@ -35,10 +35,16 @@ jobs:
         run: |
           source $VENV
           make typecheck
-      - name: Test with pytest
+      - name: Test with pytest (with coverage)
+        if: matrix.python-version != '3.11.0-beta.4'
         run: |
           source $VENV
           pytest tests -v --cov=./rich --cov-report=xml:./coverage.xml --cov-report term-missing
+      - name: Test with pytest (no coverage)
+        if: matrix.python-version == '3.11.0-beta.4'
+        run: |
+          source $VENV
+          pytest tests -v
```

Comment:
Using `--cov` causes a silent failure on 3.11, so we run without coverage on that version for now.